quark/subproject.py

```diff
@@ -239,6 +258,19 @@ def set_local_ignores(self, subprojects_dir, modules):
         pass
 
 class GitSubproject(Subproject):
+    @staticmethod
+    def get_version_control():
+        return "git"
```

Comment:
This should probably go into `get_env_variables`, and `get_version_control` can die.

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/ee6e41ed5393e010b8dd24de62bbfe021cc50f09